docs/MigrationGuide.md

```diff
@@ -26,6 +26,7 @@ Additionally the following things have changed:
 - `ecs_query_desc_t::group_by` has been renamed to `ecs_query_desc_t::group_by_callback`
 - `ecs_query_desc_t::order_by_component` has been renamed to `ecs_query_desc_t::order_by`
 - `ecs_query_desc_t::order_by` has been renamed to `ecs_query_desc_t::order_by_callback`
+- `ecs_query_desc_t::filter::terms` has been moved to `ecs_query_desc_t::terms`
```

Comment:
Change to: "`ecs_query_desc_t::filter` no longer exists, its fields (where applicable) have been moved to `ecs_query_desc_t`"